<launch>
  <param name="robot_description" textfile="$(find bar-bot)/urdf/basic_arm.urdf"/>
  <!-- <param name="robot_description" textfile="$(find bar-bot)/urdf/test.urdf"/> -->

  <node name="hebiros_node" pkg="hebiros" type="hebiros_node" respawn="false" output="screen"/>

  <!-- <node name="sn" pkg="simple" type="simple_node" respawn="false" output="screen"/> -->

  <node name="rsp" pkg="robot_state_publisher" type="robot_state_publisher">
    <!-- <remap from="joint_states" to="/hebiros/all/feedback/joint_state"/> -->
    <param name="ignore_timestamp" value="true"/>
  </node>

  <node name="jsp" pkg="joint_state_publisher" type="joint_state_publisher">
    <param name="use_gui" value="true"/>
  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find bar-bot)/rviz/viewrobot.rviz"/>
</launch>
